diff --git a/base/BUILD.gn b/base/BUILD.gn
index c438696609b40be138bee7d94f2e51f8fe53351f..592366d3e0ebf5ac3550edd8f872893b9b692c54 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5223,6 +5223,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5923,6 +5924,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
