diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index c1bbcce5dbee47b22878319281758c5dde835bfb..1295520cfd969734b3bc633419d59077c30e274e 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -1407,3 +1407,4 @@ if (enable_screen_capture) {
     "java/src/org/chromium/chrome/browser/media/TabSharingUiManager.java",
   ]
 }
+chrome_java_sources -= ["java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPageView.java",]
