diff --git a/chrome/browser/ui/omnibox/omnibox_popup_view.h b/chrome/browser/ui/omnibox/omnibox_popup_view.h
index 1cef607764d99e1434adb0dde9fe25c834ba4f3f..e4791a9ebb92daacf912e689f3dfe79e4b0ed358 100644
--- a/chrome/browser/ui/omnibox/omnibox_popup_view.h
+++ b/chrome/browser/ui/omnibox/omnibox_popup_view.h
@@ -112,6 +112,7 @@ class OmniboxPopupView {
   base::TimeTicks construction_time() const { return construction_time_; }
 
  protected:
+  BRAVE_OMNIBOX_POPUP_VIEW_FRIEND
   friend class OmniboxResultView;
   friend class OmniboxSuggestionButtonRowView;
 
