diff --git a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java
index 531c3bef4f9e73db69a1d87b284ce159e7df9aed..8e6912a2d6233078ad8ea8f2672610e438823cdd 100644
--- a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java
+++ b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java
@@ -591,7 +591,7 @@ public class TabGridDialogMediator
                                 Set.of(TabListEditorItemSelectionId.createTabId(tabId)));
                     };
             mTabGridContextMenuCoordinator =
-                    new TabGridContextMenuCoordinator(
+                    new BraveTabGridContextMenuCoordinator(
                             activity,
                             tabBookmarkerSupplier,
                             profile,
