diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
index 56695900fd80bd43c0e361c8c9878b081404e773..54aa2e896fdc84746544697fa90a7ee48ea3f027 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
@@ -204,7 +204,7 @@ import org.chromium.chrome.browser.tab_bottom_sheet.TabBottomSheetManagerImpl;
 import org.chromium.chrome.browser.tab_bottom_sheet.TabBottomSheetUtils;
 import org.chromium.chrome.browser.tab_group_suggestion.toolbar.GroupSuggestionsButtonController;
 import org.chromium.chrome.browser.tab_group_suggestion.toolbar.GroupSuggestionsButtonControllerFactory;
-import org.chromium.chrome.browser.tab_group_sync.TabGroupSyncControllerImpl;
+import org.chromium.chrome.browser.tab_group_sync.BraveTabGroupSyncControllerImpl;
 import org.chromium.chrome.browser.tab_group_sync.TabGroupSyncFeatures;
 import org.chromium.chrome.browser.tab_group_sync.TabGroupSyncServiceFactory;
 import org.chromium.chrome.browser.tab_ui.TabContentManager;
@@ -1523,7 +1523,7 @@ public class TabbedRootUiCoordinator extends RootUiCoordinator {
             var tabGroupSyncService = TabGroupSyncServiceFactory.getForProfile(originalProfile);
             assert tabGroupSyncService != null;
             mTabGroupSyncController =
-                    new TabGroupSyncControllerImpl(
+                    new BraveTabGroupSyncControllerImpl(
                             mTabModelSelectorSupplier.asNonNull().get(),
                             tabGroupSyncService,
                             UserPrefs.get(originalProfile),
