diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
index 8aa833713b8149343dd0c66a7ae84445da370ddc..4cc1bf4a73a68b8708e34acb5af7dd7abc8f5ce5 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
@@ -979,6 +979,14 @@ public class ChromeTabbedActivity extends ChromeActivity implements PreAttachInt
 
                         private void closeIfNoTabsAndHomepageEnabled(
                                 boolean isPendingClosure, boolean shouldRemoveWindowWithZeroTabs) {
+                            if (getTabModelSelector().getTotalTabCount() == 0) {
+                                shouldRemoveWindowWithZeroTabs = shouldRemoveWindowWithZeroTabs && org.chromium.chrome.browser.partnercustomizations.CloseBraveManager.shouldCloseAppWithZeroTabs();
+                            }
+                            closeIfNoTabsAndHomepageEnabled_ChromiumImpl(isPendingClosure, shouldRemoveWindowWithZeroTabs);
+                        }
+
+                        private void closeIfNoTabsAndHomepageEnabled_ChromiumImpl(
+                                boolean isPendingClosure, boolean shouldRemoveWindowWithZeroTabs) {
                             if (getTabModelSelector().getTotalTabCount() == 0) {
                                 if (shouldRemoveWindowWithZeroTabs) {
                                     mMultiInstanceManager.closeChromeWindowIfEmpty(mWindowId);
@@ -3917,6 +3925,11 @@ public class ChromeTabbedActivity extends ChromeActivity implements PreAttachInt
                 new TabModelSelectorObserver() {
                     @Override
                     public void onTabStateInitialized() {
+                        onTabStateInitialized_ChromiumImpl();
+                        onTabStateInitializedHandler();
+                    }
+
+                    private void onTabStateInitialized_ChromiumImpl() {
                         if (mMultiInstanceManager != null) {
                             mMultiInstanceManager.onTabStateInitialized();
                         }
