diff --git a/components/history/core/browser/BUILD.gn b/components/history/core/browser/BUILD.gn
index 467b4df97dcfd6a02628e4ea300104b1708ee5d3..00a4906b4bc4292a9e006dc862dea35fa65b2031 100644
--- a/components/history/core/browser/BUILD.gn
+++ b/components/history/core/browser/BUILD.gn
@@ -159,6 +159,7 @@ static_library("browser") {
     "//ui/gfx",
     "//url",
   ]
+  import("//brave/components/history/core/browser/sources.gni") sources += brave_components_history_core_browser_sources deps += brave_components_history_core_browser_deps
 }
 
 source_set("unit_tests") {
