diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 31280cc48df87f94da7fbe07eb0468c234ae40c0..eca1f81c172a117de7aca048229e468ba5759944 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3627,6 +3627,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
