diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 81c77263407991b3fd3929e57200db4f8886fc1e..8e224f88b11ccb2c392c1c86a1ae5f064ce1b408 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -782,6 +782,7 @@ source_set("browser_sources") {
   if (use_clang_profiling_inside_sandbox) {
     sources += [ "profiling_utils.h" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
