diff --git a/crypto/unexportable_key_win.cc b/crypto/unexportable_key_win.cc
index d2c89d9dd2cb89fa23d91d136615da56987a2edb..35e8ee63b4ef156875af2dd80a354fb1d9705c38 100644
--- a/crypto/unexportable_key_win.cc
+++ b/crypto/unexportable_key_win.cc
@@ -271,6 +271,8 @@ WindowsSrkHandle GetSrkHandleFor(sign::SignatureKind algo) {
     case sign::MLDSA_65:
     case sign::MLDSA_87:
       NOTREACHED();
+    case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA384:
+      return WindowsSrkHandle::kEcc;
   }
 }
 
@@ -858,6 +860,8 @@ constexpr std::optional<hash::HashKind> ToHashKind(
     case sign::MLDSA_65:
     case sign::MLDSA_87:
       return std::nullopt;
+    case SignatureVerifier::ECDSA_SHA384:
+      return hash::kSha384;
   }
 }
 
