diff --git a/DEPS b/DEPS
index c046e6935b923fde0a855573c9e40ab61e4e8f18..67a03b2cddb4e18f34b0dcadbb510915a608be99 100644
--- a/DEPS
+++ b/DEPS
@@ -4532,7 +4532,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
