diff --git a/net/BUILD.gn b/net/BUILD.gn
index 3f4f4516db26cd9ba0ded6a4f71a4f1980005f79..41491e2847565b8350e9a1790fd5b96a58b40b45 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1295,6 +1295,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
