diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 67cfca8e3ade96c8255458058cb83e21164f1bc9..720a37918201fab47574e832efa4360617d19daa 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3450,6 +3450,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
